tests/trainer/test_trainer.py

```diff
@@ -1128,9 +1129,16 @@ def test_num_sanity_val_steps(tmpdir, limit_val_batches):
 def test_num_sanity_val_steps_neg_one(tmpdir, limit_val_batches):
     """Test that `num_sanity_val_steps=-1` runs through all validation data once, and as many batches as limited by
     `limit_val_batches` Trainer argument."""
-    model = EvalModelTemplate()
-    model.validation_step = model.validation_step__multiple_dataloaders
-    model.validation_epoch_end = model.validation_epoch_end__multiple_dataloaders
+
+    class CustomModel(BoringModel):
+        def validation_step(self, batch, batch_idx, dataloader_idx):
```

Comment:
I think we could add `dataloader_idx=0` to BoringModel so this isn't necessary in each subclass